CONTRIBUTING.md

```diff
@@ -15,6 +15,54 @@ propose changes to this document in a pull request.
 * Perform a [cursory search](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+repo%3Aprojectatomic%2Fatomicapp)
   to see if a similar issue has already been submitted
 
+## Developers
+
+First of all, fork the [repository](https://github.com/projectatomic/atomicapp) and clone your github repository:
```

Comment:
`Fork [atomicapp](https://github.com/projectatomic/atomicapp) and clone your GitHub repo:`